app/validators/external_vacancy_validator.rb

```diff
@@ -8,12 +8,29 @@ def validate(record)
     )
 
     record.errors.add(:organisations, "No school(s) associated with vacancy") if record.organisations.empty?
+    validate_uniqueness_per_client(record)
+    validate_no_duplicate_vacancy(record)
     validate_job_title_length(record)
-    validate_expiry_date(record) if record.expires_at.present?
```

Comment:
I prefer the original pattern rather than the short-circuit in the method - feels clearer that we only validate the expiry date if it exists, rather than the short-return (which I thought we agreed not to use on small methods) on the method itself

Reply:
I treated it like any other method, where I would guard it against giving the presence of the `expires_at` for granted could cause 
```ruby
nil < Time.zone.today
NoMethodError: undefined method '<' for nil (NoMethodError)
```
But you're right, in a validation context where we want to apply a validation, the original form was clearer about when it runs.